my-app/src/components/Docs/FileUploader.js

```diff
@@ -19,22 +21,24 @@ class FilesUploader extends Component {
       progress: 0
     })
   }
-
+  
+  
   handleUploadSuccess = filename => {
     this.setState({
       image: filename,
       progress: 100
     })
 
+
     firebase.storage().ref('avatars').child(filename).getDownloadURL()
       .then(url => this.setState({
```

Comment:
you should also call callback from props to emit this data to parent.
sth like this

```jsx
firebase.storage().ref('avatars').child(filename).getDownloadURL()
      .then(url => {
        this.setState({
          imageURL: url
        });
        this.props.fileSaved(url);
      })
```